lib/openstates/api.rb

```diff
@@ -15,6 +15,11 @@ def bills(options = {})
         end
       end
 
+      if options[:id]
+        url << "#{options[:id]}/"
+        options.delete(:id)
+      end
```

Comment:
They have two different endpoints to get a bill's details:

```/bills/state/session/bill_id/``` and ```/bills/id```

The first uses the bill_id attribute, which is "The official id of the bill (e.g. 'SB 27', 'A 2111')" according to them. The second endpoint uses the id attribute, which is: "Open States-assigned permanent ID for this bill.".

The truth is that I couldn't get any detail on the first endpoint, only on the second one. But I left the first one there because it's documented.